src/sbd-inquisitor.c

```diff
@@ -56,6 +60,22 @@ void recruit_servant(const char *devname, pid_t pid)
 	newbie->pid = pid;
 	newbie->first_start = 1;
 
+	/* some sanity-check on our newbie */
+	if (sbd_is_disk(newbie)) {
+	    cl_log(LOG_INFO, "Monitoring %s", devname);
+	    disk_count++;
```

Comment:
Unrelated to this PR, but maybe a comment here would be useful as to why outdated is set below but not above